packages/core/src/components/map/Map.tsx

```diff
@@ -63,69 +63,68 @@ interface PrivateMapProps<Type extends Area> extends MapProps<Type> {
   areas: readonly Type[]
 }
 
-const defaultProps: MapProps<Area> = {
-  style: {},
-  color: '#ccc',
-  showTooltip: true,
-  hoverable: true,
-  zoomable: true
-}
-
-function Map<Type extends Area>(props: PrivateMapProps<Type>) {
+const Map = <Type extends Area>({
+  style = {},
+  color = '#ccc',
+  showTooltip = true,
+  hoverable = true,
+  zoomable = true,
+  ...props
+}: PrivateMapProps<Type>) => {
   test.rerenders()
+  const {
+    areas,
+    clickable,
```

Comment:
Exactly, and we should be able to move the default parameters into the destructuring of the props from line 76, i.e.:

```ts
const = {
  style = {},
  ....,
  areas,
  ...
}
```